src/components/expanded-state/unique-token/saveToCameraRoll.js

```diff
@@ -23,9 +26,9 @@ const getPermissionAndroid = async () => {
 
 function alertError(err) {
   Alert.alert(
-    'Failed to save Image',
+    lang.t('expanded_state.unique.save.failed_to_save_image'),
```

Comment:
This is not a change, but the casing seems weird here to me of the original text "Failed to save Image" @skylarbarrera @christianbaroni did we want to capitalize all or lowercase the rest of the words?